crates/sui-types/src/messages.rs

```diff
@@ -2461,6 +2461,21 @@ pub enum VerifiedTransactionInfoResponse {
     ),
 }
 
+#[derive(Clone, Debug)]
+pub enum PlainTransactionInfoResponse {
```

Comment:
How is this different from `TransactionInfoResponse` in terms of the information it contains?

Reply:
I'd suggest to use generics here to avoid duplication

Reply:
ideally we can remove `enum VerifiedTransactionInfoResponse`

Reply:
i see, TransactionInfoResponse should be sufficient here. refactoring.